Interfaces/API/Job.py

```diff
@@ -554,26 +556,83 @@ def setDestination(self, destination):
     return S_OK()
 
   #############################################################################
-  def setNumberOfProcessors(self, processors):
+  def setNumberOfProcessors(self, processors=None, minNumberOfProcessors=None, maxNumberOfProcessors=None):
     """Helper function.
 
        Example usage:
 
        >>> job = Job()
-       >>> job.setNumberOfProcessors(4)
+       >>> job.setNumberOfProcessors(2)
+       means that the job needs 2 processors
```

Comment:
We have keyed function arguments. I think we said that for keyed arguments in a function call we should always give keys explicitly. So, it is better to reflect it in the function documentation also. E.g. :
   job.setNumberOfProcessors(processors = 2)
   job.setNumberOfProcessors(minNumberOfProcessors = 4, maxNumberOfProcessors = 8)